bears/coffee_script/CoffeeLintBear.py

```diff
@@ -25,8 +30,206 @@ class CoffeeLintBear:
+                              if end_file_with_a_single_newline else {})
+        coffee_configs.update({"newlines_after_classes":
+                               {"value": number_of_lines_after_classes,
+                                "level": "ignore"}}
+                              if newlines_after_classes else {})
+        coffee_configs.update({"no_backticks":  {"level": "error"}}
+                              if no_backticks else {})
+
+        coffee_configs.update({"no_implicit_braces":
+                               {"level": "ignore", "strict": 'true'}}
+                              if no_implicit_braces else {})
+
+        coffee_configs.update({"no_implicit_parens":
+                               {"strict": 'true', "level": "ignore"}}
+                              if no_function_call_without_parentheses else {})
+        coffee_configs.update({"no_interpolation_in_single_quotes":
+                               {"level": "ignore"}}
+                              if no_interpolation_in_single_quotes else {})
+        coffee_configs.update({"no_stand_alone_at": {"level": "ignore"}}
+                              if no_stand_alone_at else {})
+        coffee_configs.update({"no_tabs":
+                               {"level": "error"}} if no_tabs else {})
+
+        coffee_configs.update({"no_throwing_strings": {"level": "error"}}
+                              if no_throwing_strings else {})
+        coffee_configs.update({"no_trailing_semicolons":
+                               {"level": "error"}}
+                              if no_trailing_semicolons else {})
+        coffee_configs.update({"no_trailing_whitespace":
+                               {"level": "error",
+                                "allowed_in_comments": _coffee_bool(
+                                  apply_no_trailing_whitespaces_in_comments),
+                                "allowed_in_empty_lines": _coffee_bool(
+                                  apply_no_trailing_whitespaces_in_empty_line)}}
+                              if no_trailing_whitespaces else {})
+        coffee_configs.update({"no_unnecessary_double_quotes":
+                               {"level": "ignore"}}
+                              if no_unnecessary_double_quotes else {})
+        coffee_configs.update({"prefer_english_operator":
+                               {"level": "ignore", "doubleNotLevel": "ignore"}}
+                              if use_english_operator else {})
+        coffee_configs.update({"space_operators":
+                               {"level": "ignore"}}
+                              if space_around_operators else {})
+        coffee_configs.update({"spacing_after_comma":
+                               {"level": "ignore"}}
+                              if space_after_comma else {})
+        config_json = json.dumps(coffee_configs)
+        configs = config_json[1:-1]
+        return '--reporter=raw', filename, configs
```

Comment:
it sure is, i was suppose to add -f before configs [of course before the filename]